lib/migration.pm

```diff
@@ -158,7 +158,7 @@ sub disable_installation_repos {
 
 # Based on bsc#1097111, need to disable kernel multiversion before migration, and enable it after migration
 # https://documentation.suse.com/sles/15-SP3/html/SLES-all/cha-update-preparation.html#sec-update-preparation-multiversion
-sub disable_kernel_multiversion {
+sub modify_kernel_multiversion {
     my $sed_para = check_var('VERSION', get_var('ORIGIN_SYSTEM_VERSION')) ? 's/^multiversion/#multiversion/g' : 's/^#multiversion/multiversion/g';
     script_run("sed -i $sed_para /etc/zypp/zypp.conf");
 }
```

Comment:
Because this makes sense apparently only for migration flavor,  I think the condition should be moved here as well.
```
sub modify_kernel_multiversion {
if (get_var('FLAVOR', '') =~ /Continuous-Migration/) {....}
}```
wdyt?

Reply:
I agree to move condition inside function, this can make it looks more clean to invoke function.

Reply:
I wasn't going to touch the function, but since you guys think it is within the scope of my ticket, I will remove this condition in sed, and replace it with (enable/disable) arguments which will make it more clear and will make the function usable for other tests besides migration.

Reply:
Also, I don't think we should spend time for refactoring minor parts and running VRs for these modules that should wholly be refactored